### PR TITLE
add purchase conflict test

### PR DESCRIPTION
**Pull request by OrenZak**
- added test for purchase API, check that conflict is handled and return the order confirmation.
- changed the amount of the native offer on sample app, as requested.

### CODE REVIEW

File: kin-ecosystem-sdk/src/test/java/com/kin/ecosystem/data/order/OrderRepositoryTest.java [hunk 1]
```diff
@@ -330,6 +339,54 @@ public void onFailure(Throwable t) {
...
+
+        when(remote.createExternalOrderSync(anyString())).thenThrow(apiException);
+        when(remote.getOrderSync(anyString())).thenReturn(confirmedOrder);
+        when(offerRepository.getPendingOffer()).thenReturn(pendingOffer);
+
+        // Check not error, and got jwt confirmation
+        orderRepository.purchase("A GENERATED NATIVE OFFER JWT", new Callback<OrderConfirmation>() {
+            @Override
+            public void onResponse(OrderConfirmation orderConfirmation) {
+                countDownLatch.countDown();
```
Comment (YohayBar) [inline kin-ecosystem-sdk/src/test/java/com/kin/ecosystem/data/order/OrderRepositoryTest.java]: countDownLatch.countDown(); in the end

File: kin-ecosystem-sdk/src/test/java/com/kin/ecosystem/data/order/OrderRepositoryTest.java [hunk 2]
```diff
@@ -330,6 +339,54 @@ public void onFailure(Throwable t) {
...
+                assertEquals("A JWT CONFIRMATION", orderConfirmation.getJwtConfirmation());
```
Comment (yosriz) [inline kin-ecosystem-sdk/src/test/java/com/kin/ecosystem/data/order/OrderRepositoryTest.java]: If I recall correctly there might be a problem asserting not in the main (tests) thread

File: kin-ecosystem-sdk/src/test/java/com/kin/ecosystem/data/order/OrderRepositoryTest.java (hunk 1 above)
Comment (OrenZak) [reply kin-ecosystem-sdk/src/test/java/com/kin/ecosystem/data/order/OrderRepositoryTest.java]: ?

File: kin-ecosystem-sdk/src/test/java/com/kin/ecosystem/data/order/OrderRepositoryTest.java (hunk 2 above)
Comment (OrenZak) [reply kin-ecosystem-sdk/src/test/java/com/kin/ecosystem/data/order/OrderRepositoryTest.java]: this is why we have the countDownLatch

**Pull request merged**